gpii/node_modules/flowManager/src/CloudBasedFlowManager.js

```diff
@@ -212,3 +212,14 @@ gpii.flowManager.cloudBased.oauth2settings.handleRequest = function (request, ha
         request.events.onError.fire(gpii.oauth2.errors.unauthorized);
     });
 };
+
+gpii.flowManager.cloudBased.matchToUntrustedSettings = function (finalPayload, event) {
+    // Note that we send the filtered preferences to assist a PCP - we may do more filtering here at some point
```

Comment:
Recall that it is now a "PSP" .... explain more here about what is being done and why.